contrib/mesos/pkg/scheduler/constraint/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package constraint exposes Marathon-like constraints for scheduling pods.
+// Incomplete.
```

Comment:
I think a better approach would be to see if we can just make the Kubernetes constraint language somehow compatible with Mesos one. We really want to avoid having Kubernetes-on-Mesos have a different API than native Kubernetes, and stuffing constraints into label for k8sm is a pretty bad abuse of the API. We haven't really figured out what we want for the full label selector syntax in Kubernetes yet (see #367) so this is a good thing to keep in mind as we work on it.